httpserver/service_node.h

```diff
@@ -85,6 +89,8 @@ class ServiceNode {
     /// map pubkeys to a list of connections to be notified
     std::unordered_map<pub_key_t, listeners_t> pk_to_listeners;
 
+    const loki::lokid_key_pair_t& lokid_key_pair_;
```

Comment:
This allows me to forward declare `lokid_key_pair_t` instead of #including it.
Also, wasn't sure if `std::array` was copyable, which it seems to be. So changing its own copy for code simplicity.

Reply:
It is a small header that's unlikely to change often, so should be fine.

Reply:
Changed.